src/ValueResolver/NodeExpressionResolver.php

```diff
@@ -256,7 +255,12 @@ protected function resolveExprConstFetch(Expr\ConstFetch $node)
 
     protected function resolveExprClassConstFetch(Expr\ClassConstFetch $node)
     {
-        $refClass     = $this->fetchReflectionClass($node->class);
+        $classToReflect = $node->class;
```

Comment:
Anything that would force PHP-Parser to create `Expr` class would do. Code won't be ever evaluated by PHP itself and only needs to be syntactically valid.